<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
    <horstretch>55</horstretch>
    <verstretch>15</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>1600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
 background-color :  rgb(0, 140,150);
	color:  rgb(0, 140,150);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QMainWindow{
		 background-color: rgb(0, 0, 0);
}</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>1251</width>
      <height>701</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1900</width>
      <height>1600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView{
 background :  rgb(0, 140,150);
}</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>270</y>
      <width>471</width>
      <height>302</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>25</number>
     </property>
     <property name="verticalSpacing">
      <number>16</number>
     </property>
     <item row="2" column="1">
      <widget class="QLineEdit" name="targetSpeedEdit">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="CourseLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Target X position</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="ktsLabel_3">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color :rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>kts</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="deg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>15</number>
       </property>
       <property name="text">
        <string>deg</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="CourseLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Target Course</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="targetSpeedLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Target Speed</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="targetX">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="ktsLabel_4">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color :rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>m</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="ktsLabel">
       <property name="toolTipDuration">
        <number>5</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color :rgb(0, 140,150);
}</string>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="midLineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>kts</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="ktsLabel_2">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color :rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>sec</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="speedInput">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Add Target</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="targetY">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="CourseLabel_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Target Y position</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="periodEdit_2">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="SpeedName">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Torpedo Speed</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="angleInput">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="CourseLabel_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>Time period</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="ktsLabel_5">
       <property name="styleSheet">
        <string notr="true">QLabel{
 background-color :  rgb(10, 10,10);
	color :rgb(0, 140,150);
}</string>
       </property>
       <property name="text">
        <string>m</string>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>680</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="hostCombo">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>50</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="hostLabel">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>Server name</string>
    </property>
    <property name="buddy">
     <cstring>hostCombo</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server port</string>
    </property>
    <property name="buddy">
     <cstring>portLineEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>100</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="courseAngleLabel">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>150</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Torpedo Angle (deg)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabelKTS">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Speed (kts)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="angleData">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedData">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shutDown">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>680</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>10</red>
          <green>10</green>
          <blue>10</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>140</green>
          <blue>150</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
    </property>
    <property name="text">
     <string>Shut Down</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>580</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background-color :  rgb(10, 10,10);
	color : rgb(0, 140,150);
}</string>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QLabel" name="periodLabel">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>210</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Time Period(sec)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="periodData">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>791</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Torpedo Status -&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <family>Courier 10 Pitch</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>-5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>changeButton</zorder>
   <zorder>graphicsView</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>hostCombo</zorder>
   <zorder>hostLabel</zorder>
   <zorder>portLabel</zorder>
   <zorder>portLineEdit</zorder>
   <zorder>courseAngleLabel</zorder>
   <zorder>speedLabelKTS</zorder>
   <zorder>angleData</zorder>
   <zorder>speedData</zorder>
   <zorder>shutDown</zorder>
   <zorder>setButton</zorder>
   <zorder>periodLabel</zorder>
   <zorder>periodData</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
